common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -841,4 +841,5 @@ describe 'Problem', ->
       jasmine.clock().tick(64000)
       expect(@problem.poll.calls.count()).toEqual(6)
 
-      expect($('.capa_alert').text()).toEqual("The grading process is still running. Refresh the page to see updates.")
+      expect($('.notification-gentle-alert .notification-message').is(':visible')).toBe(true)
```

Comment:
I need to refactor this test, it looks like once I removed the hide or the style display: none it stopped working.